sos/plugins/openstack_instack.py

```diff
@@ -28,6 +30,16 @@ def setup(self):
         self.add_copy_spec("/home/stack/instackenv.json")
         self.add_copy_spec("/home/stack/undercloud.conf")
 
+        # collect undercloud-debug: undercloud-debug.log will be created so we
+        # need to generate it in a tmp.dir, collect it now and then cleanup it
+        tmpdir = gettempdir()
+        self.get_cmd_output_now("su -s /bin/sh -c 'source ~/stackrc && " +
+                                "/usr/libexec/openstack-tripleo/" +
+                                "undercloud-debug' stack",
+                                timeout=600, runat=tmpdir,
+                                suggest_filename="undercloud-debug.log")
+        remove("%s/undercloud-debug.log" % tmpdir)
```

Comment:
The requirements came from RHBZ1250346 where they might not reflect what sos already collects.

@lyarwood could you please provide list of tripleo specific commands worth to be collected, instead of whole undercloud-debug? That would be the best solution here. Also please specify if `source ~/stackrc` is required for them (see BZ discussion).